libraries/AP_Scheduler/AP_Scheduler.cpp

```diff
@@ -389,6 +404,51 @@ void AP_Scheduler::Log_Write_Performance()
     AP::logger().WriteCriticalBlock(&pkt, sizeof(pkt));
 }
 
+// display task statistics as text buffer for @SYS/tasks.txt
+size_t AP_Scheduler::task_info(char *buf, size_t bufsize) const
+{
+    size_t total = 0;
+
+    // a header to allow for machine parsers to determine format
+    int n = hal.util->snprintf(buf, bufsize, "TasksV1\n");
+
+    if (n <= 0 || perf_info.get_task_info(0) == nullptr) {
```

Comment:
Still the silent failure here when the thing is disabled?

Reply:
Fixed and dynamic enabled implemented